spec/rubocop/cop/rspec/variable_name_spec.rb

```diff
@@ -68,6 +72,18 @@
           let("user#{name}") { 'Adam' }
         RUBY
       end
+
+      it 'does not register offense for configured ignoredVariable' do
+        expect_no_offenses(<<~'RUBY')
+          let('ignoredVariable') { 'Adam' }
+        RUBY
+      end
```

Comment:
Please add those examples separate from `context 'when configured for `snake_case`' do`, along with config setup, and preferably each having a balancing example where the same variable name is flagged.